src/TableUtils.jl

```diff
@@ -18,15 +34,25 @@ function format_time(val::Dict)
         @sprintf("%.3g %s", val["median"] * unit, unit_name)
     end
 end
-function format_time(val::Number)
-    unit, unit_name = get_reasonable_memory_unit([val])
+
+function format_time(
+    val::Number, row_name::String=""; time_unit::Union{Nothing,String}=nothing
+)
+    if time_unit === nothing
+        unit, unit_name = get_reasonable_memory_unit([val])
+    else
+        unit, unit_name = get_time_unit_scale(time_unit)
+    end
     @sprintf("%.3g %s", val * unit, unit_name)
 end
-function format_time(::Missing)
+
+function format_time(
+    ::Missing, row_name::String=""; time_unit::Union{Nothing,String}=nothing
+)
     return ""
 end
 
-function format_memory(val::Dict)
+function format_memory(val::Dict, _row_name::String="")
```

Comment:
What's this?

Reply:
`default_formatter`  takes `(val, row_name)` as arguments. That way the logic at https://github.com/MilesCranmer/AirspeedVelocity.jl/pull/123/files#diff-16858931f1973bf53ef24ef190368d8b5dc9d99885089da1178ac8dcb65434faR84-R88 can make `"time_to_load"` as special case that does not take formatting.